pages/blog/async-api-spec-3.0-release.md

```diff
@@ -0,0 +1,59 @@
+-   [Address perspective and channel reuse issues through introducing 'endpoint' concept](https://github.com/asyncapi/spec/issues/599)
+
+The tooling issues and pull requests which are tied up with the Spec 3.0 release are:
+
+-   [Implement intent-driven Parser API](https://github.com/asyncapi/parser-js/issues/401)
+-   [Splitting out definitions into separate files](https://github.com/asyncapi/spec-json-schemas/issues/127)
+-   [Move binding JSON schema files to main JSON schema repository](https://github.com/asyncapi/bindings/issues/113)
+
+We are using the [GitHub milestone](https://github.com/asyncapi/spec/milestone/18) to track down all the issues related to Spec 3.0 release. If you are willing to take up an issue, filter the issues with `needs champion` label and drive the initiative forward. We also have a [release journal](https://github.com/asyncapi/community/issues/163) for the upcoming Spec 3.0 release to keep note of all the references, build the right documentation, and track the progress for the upcoming release.
+
+## Contributing to the release
+
+AsyncAPI is a community-driven project and we welcome all kinds of contributions! To get started, join our [Slack workspace](http://asyncapi.com/slack-invite). After joining, reach out to us on the `#03_specification` channel and participate in the conversations.
+
+To better understand AsyncAPI and how the Spec 3.0 release will play out, we recommend watching our [past meeting recordings](https://www.youtube.com/watch?v=CLNgLB4-UnA&list=PLbi1gRlP7pihClJY-kXuTRRJ8n1awb0VV). To know how you can contribute, we recommend checking out our [contributing guidelines](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md). If you wish to keep track of all the work that is happening on the Spec 3.0 release, we recommend checking out our [GitHub issue](https://github.com/asyncapi/spec/issues/691).
+
+Before landing in your contributions, make sure you follow our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and the pull request you raised is [described properly](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits). To know how we collaborate using Git & GitHub, follow the [Git workflow](https://github.com/asyncapi/.github/blob/master/git-workflow.md) that we have developed for new contributors.
+
+## When will the release happen?
+
+We are yet to arrive at a date but the community is working towards releasing this by the end of 2022. Currently, we meet every two weeks to discuss the upcoming Spec 3.0 release and share updates and you are welcome to join it! We are going to have our next meeting on [UTC 16:00 on March 3, 2022](https://github.com/asyncapi/community/issues/270). You can access the [AsyncAPI Calendar](https://calendar.google.com/calendar/u/0/embed?src=tbrbfq4de5bcngt8okvev4lstk@group.calendar.google.com) and explore our upcoming meetings.
```

Comment:
I guess we have to link the next meeting here, @derberg do you mind pinging us when the new issue is up?

Reply:
@jonaslagoni actually you can create it as you have proper rights in repo. Just go to https://github.com/asyncapi/community/actions/workflows/create-event-issue.yml an click Run workflow button

but first you need to update l with zoom and streaming info. It proved to be good, so we can make it official there. For now I was creating issue and editing it manually

Reply:
@HarshCasper there you go: https://github.com/asyncapi/community/issues/282